Loenn/lang/en_gb.lang

```diff
@@ -0,0 +1,133 @@
+entities.FactoryHelper/Conveyor.placements.name.start_right=Conveyor (Right)
+entities.FactoryHelper/Conveyor.attributes.description.activationId=String value of the entity's Activation Id. When a factory activator with this Id is turned on the entity will toggle state.
+entities.FactoryHelper/Conveyor.attributes.description.startLeft=If `true` the conveyor will initially be moving left.
+
+# Crate
+entities.FactoryHelper/ThrowBox.placements.name.wood=Crate (Wood)
+entities.FactoryHelper/ThrowBox.placements.name.metal=Crate (Metal)
+entities.FactoryHelper/ThrowBox.attributes.description.isMetal=Whether the crate is metallic or not. If `false` it will be wooden.
+entities.FactoryHelper/ThrowBox.attributes.description.tutorial=Whether the tutorial messsages should show up.
+entities.FactoryHelper/ThrowBox.attributes.description.isSpecial=Whether the crate should kill the player and return them to the crate's origin upon destruction or player death.
+entities.FactoryHelper/ThrowBox.attributes.description.isCrucial=Whether the crate should kill the player upon destruction.
+
+# Crate Spawner
+entities.FactoryHelper/ThrowBoxSpawner.placements.name.crate_spawner=Crate Spawner
+entities.FactoryHelper/ThrowBoxSpawner.attributes.description.activationId=String value of the entity's Activation Id. When a factory activator with this Id is turned on the entity will toggle state.
+entities.FactoryHelper/ThrowBoxSpawner.attributes.description.startActive=If `true` the entity will initially be active.
+entities.FactoryHelper/ThrowBoxSpawner.attributes.description.delay=The amount if time before a new crate is spawned after the previous one in seconds.
+entities.FactoryHelper/ThrowBoxSpawner.attributes.description.isMetal=Whether the crates spawned are metallic or not. If `false` they will be wooden. Ignored if Is Random is true.
+entities.FactoryHelper/ThrowBoxSpawner.attributes.description.isRandom=Whether the spawned crates should have a random material. If set to `true` the `Is Metal` value will be ignored.
+entities.FactoryHelper/ThrowBoxSpawner.attributes.description.fromTop=If `true` they will spawn on top, otherwise they will spawn at the nearest side. Defautls to `true`.
+entities.FactoryHelper/ThrowBoxSpawner.attributes.description.tutorial=Whether the tutorial messsages should show up.
+entities.FactoryHelper/ThrowBoxSpawner.attributes.description.maximum=The maximum number of crate this spawner can spawn. Destroyed crates do not count towards the maximum.
+
+# Dash Fuse Bose
+entities.FactoryHelper/DashFuseBox.placements.name.left=Dash Fuse Box (Left)
+entities.FactoryHelper/DashFuseBox.placements.name.right=Dash Fuse Box (Right)
+entities.FactoryHelper/DashFuseBox.attributes.description.direction=The direction the fuse box is facing.
+entities.FactoryHelper/DashFuseBox.attributes.description.activationIds=A comma separated set of string Ids for all the Activation Ids this box should activate.
```

Comment:
might want to edit the tooltip to mention that no activation ID will make the fuse box default to the broken state? or could maybe add a validator function to force one to be entered (unless someone uses these for aesthetics lol)